sale_commission/wizard/wizard_settle.py

```diff
@@ -63,7 +63,7 @@ def action_settle(self):
             date_to_agent = self._get_period_start(agent, date_to)
             # Get non settled invoices
             agent_lines = agent_line_obj.search(
-                [('invoice_date', '<', date_to_agent),
+                [('invoice_date', '<', date_to),
```

Comment:
No, it isn't. You can't make the settlement of something in progress, as it's still not completed (there are possible invoices to come).